### ISSUE TITLE
Have the plugin adhere to exact key combinations

### ISSUE THREAD

**Issue by apa20211** [neilrush/Camera-Keys#12]
It would be nice if the keybinds weren't able to be activated if other modifier keys were used.

Ex:
If I have 'W' bound to North, CTRL+W shouldn't activate the keybind.

**Comment by JoshPaulie** [comment]
I agree, but for an opposite reason. I'd like Ctrl+A to face west, while A still pans left

**Comment by blankaex** [comment]
+1 for supporting modifier keys. I suspect WASD for regular camera movement and WASD with a modifier for NESW would be a common conclusion.

**Comment by LouisLP** [comment]
> +1 for supporting modifier keys. I suspect WASD for regular camera movement and WASD with a modifier for NESW would be a common conclusion.

I had the exact same thought, would be so convenient to have:

- `Ctrl`+`W` for North
- `Ctrl`+`D` for East
- `Ctrl`+`S` for South
- `Ctrl`+`A` for West

Any plans on including this @neilrush
